Jenkinsfile

```diff
@@ -104,14 +129,20 @@ pipeline {
 				}
 			}
 		}
-		// This stage archives all logs and database backups produced by UpdateStableIdentifiers
+		// This stage archives all logs and database backups produced by UpdateStableIdentifiers in the Reactome s3 bucket.
 		stage('Archive logs and backups'){
 			steps{
 				script{
-					sh "mkdir -p archive/${currentRelease}/logs"
-					sh "mv --backup=numbered *_${currentRelease}_*.dump.gz archive/${currentRelease}/"
+					def s3Path = "${env.S3_RELEASE_DIRECTORY_URL}/${currentRelease}/update_stable_ids"
+					sh "mkdir -p databases/"
+					sh "mv --backup=numbered *_${currentRelease}_*.dump.gz databases/"
+					sh "mv data-release-pipeline/ortho-stable-id-history/logs/* logs/"
+					sh "rm -rf data-release-pipeline*"
 					sh "gzip logs/*"
-					sh "mv logs/* archive/${currentRelease}/logs/"
+					sh "aws s3 --no-progress --recursive cp databases/ $s3Path/databases/"
```

Comment:
Does `--recursive` cause the nested directory structures to be recreated in the same way in the S3 bucket?

Reply:
Yes it does, with the exception of the highest-level folder. So if you have directory structure folder1>folder2>folder3 and you execute the command `aws s3 cp --recursive folder1/ s3://some/destination/`, you will get all of the contents of folder1, but not folder1 itself. So the final structure would look like `s3://some/destination/folder2/folder3`